Core/GDCore/IDE/ExportedDependencyResolver.h

```diff
@@ -14,39 +14,41 @@ namespace gd {
  * \note Both objects must be kept alive, as this is keeping a pointer to them.
  */
 struct DependencyMetadataAndExtension {
-  DependencyMetadataAndExtension(gd::DependencyMetadata& dependency_,
-                                 gd::PlatformExtension& extension_)
+  DependencyMetadataAndExtension(gd::DependencyMetadata &dependency_,
```

Comment:
Is this the result of running clang-format? If so I can run clang-format in master to avoid this noise.

Reply:
Ah yes I recently changed the IDE formatter from the builtin one from the VSCode C/C++ extension to the clang-format extension (as it was in the repository recomended extensions)